cmd/kepler/main.go

```diff
@@ -105,3 +112,33 @@ func waitForInterrupt(ctx context.Context, signals ...os.Signal) (func() error,
 			cancel()
 		}
 }
+
+func parseArgsAndConfig() (*config.Config, error) {
+	app := kingpin.New("power-monitor", "Power consumption monitoring exporter for Prometheus.")
+
+	configFile := app.Flag("config.file", "Path to YAML configuration file").String()
```

Comment:
add a default path same as what was for original kepler

Reply:
Default path indicates that the user has provided a file even when they haven't. I think that is a wrong assumption to make.

Reply:
Expanding on the above ... say the invocation is 

```
1. kepler
2. kepler --config.file=/path/to/a/config.file.yaml
3. kepler --config.file=/path/to/default/config.yaml
```

What should the result be if these files are missing?

Reply:
operator will be providing a config always, the path can be fixed there.  this is just a suggestion, you can ignore it.